backend/src/openarchiefbeheer/destruction/tests/test_serializers.py

```diff
@@ -111,7 +111,9 @@ def test_create_destruction_list(self):
 
         self.assertEqual(
             message,
-            _('Destruction list "%(list_name)s" created by user %(author)s.')
+            _(
+                'User "%(author)s" with the role of "" has created destruction list "%(list_name)s".'
+            )
```

Comment:
Same as backend/src/openarchiefbeheer/destruction/tests/endpoints/test_co_reviewers.py

And same for the changes below